client-mvp-04/package.json

```diff
@@ -55,18 +57,14 @@
     ]
   },
   "devDependencies": {
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest-dom": "^3.6.5",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^4.1.2",
-    "eslint-plugin-testing-library": "^3.10.1",
+    "eslint": "^6.6.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-config-airbnb": "^18.2.1",
```

Comment:
I also got 

<details>
    <summary>this error</summary>

![Screen Shot 2021-06-24 at 1 50 11 PM](https://user-images.githubusercontent.com/53061723/123346045-3fb3bd80-d50c-11eb-8470-fdbf65119fbd.png)


</details>

So i ran this command, and it worked... sort of
`npm install eslint-plugin-jsx-a11y@latest --save-dev   `

Reply:
I have deleted and installed again the dependencies. I am not getting this problem still. How did you got this error?

Reply:
I have installed this package globally. Because of that, I was not getting this error.